packages/material-ui/src/Select/Select.js

```diff
@@ -101,6 +101,8 @@ Select.propTypes = {
   classes: PropTypes.object.isRequired,
   /**
    * If `true`, the selected item is displayed even if its value is empty.
```

Comment:
```suggestion
   * If `true`, a value is displayed even if no items are selected.
```

I don't understand the use case for this prop<superscript>*</superscript>, but from the related issue it appears it is only useful along with `multiple`? If so, this description should probably say so.

<superscript>*</superscript> A) What's the purpose? B) it clashes with the floating label, but perhaps that's a bug: https://codesandbox.io/s/material-demo-w114k

Reply:
Hmm looks like a bug to me

Reply:
There is a demo with `displayEmpty` in the https://material-ui.com/components/selects/#simple-select demo. It wasn't introduced for multiple use cases initially. Yes exactly, it's useful when no floating label is used.